lib/billy.rb

```diff
@@ -42,11 +42,11 @@ def self.register_drivers
 
     if defined?(Capybara::Webkit::Driver)
       Capybara.register_driver :webkit_billy do |app|
-        driver = Capybara::Webkit::Driver.new(app)
-        driver.browser.ignore_ssl_errors
-        driver.browser.set_proxy(host: Billy.proxy.host,
-                                 port: Billy.proxy.port)
-        driver
+        options = {
+          :ignore_ssl_errors => false,
+          :proxy => {:host => Billy.proxy.host, :port => Billy.proxy.port}
+        }
+        Capybara::Webkit::Driver.new(app, Capybara::Webkit::Configuration.to_hash.merge(options))
```

Comment:
Does this allow you to specify webkit config in your rails helper?  If so, is there a race condition / order dependence of the config blocks we should note in the README?